packages/core/src/actions/smartAccount/buildUserOperationFromTx.ts

```diff
@@ -19,6 +19,81 @@ import type {
 import { buildUserOperation } from "./buildUserOperation.js";
 import type { UserOperationContext } from "./types.js";
 
+/**
+ * Performs [`buildUserOperationFromTx`](./buildUserOperationFromTx.md) in batch and builds into a single, yet to be signed `UserOperation` (UO) struct. The output user operation struct will be filled with all gas fields (and paymaster data if a paymaster is used) based on the transactions data (`to`, `data`, `value`, `maxFeePerGas`, `maxPriorityFeePerGas`) computed using the configured [`ClientMiddlewares`](/packages/aa-core/smart-account-client/middleware/index) on the `SmartAccountClient`
+ * 
+ * @example
+ * ```ts
+import type { RpcTransactionRequest } from "viem";
+import { smartAccountClient } from "./smartAccountClient";
+ * @param skipSigning
+ * @param skipSigning
+ * @param skipSigning
+ * @param skipSigning
+ * @param skipSigning
+ * @param skipSigning
+ * @param skipSigning
+ * @param skipSigning
+ * @param skipSigning
+ * @param skipSigning
+// [!code focus:99]
+// buildUserOperationFromTxs converts traditional Ethereum transactions in batch and returns
+// the unsigned user operation struct after constructing the user operation struct
+// through the middleware pipeline
+const requests: RpcTransactionRequest[] = [
+  {
+    from, // ignored
+    to,
+    data: encodeFunctionData({
+      abi: ContractABI.abi,
+      functionName: "func",
+      args: [arg1, arg2, ...],
+    }),
+  },
+  {
+    from, // ignored
+    to,
+    data: encodeFunctionData({
+      abi: ContractABI.abi,
+      functionName: "func",
+      args: [arg1, arg2, ...],
+    }),
+  },
+  ...
+  {
+    from, // ignored
+    to,
+    data: encodeFunctionData({
+      abi: ContractABI.abi,
+      functionName: "func",
+      args: [arg1, arg2, ...],
+    }),
+  },
+];
+const uoStruct = await smartAccountClient.buildUserOperationFromTxs({
+  requests,
+});
+ 
+// signUserOperation signs the above unsigned user operation struct built
+// using the account connected to the smart account client
+const request = await smartAccountClient.signUserOperation({ uoStruct });
+ 
+// You can use the BundlerAction `sendRawUserOperation` (packages/core/src/actions/bundler/sendRawUserOperation.ts)
+// to send the signed user operation request to the bundler, requesting the bundler to send the signed uo to the
+// EntryPoint contract pointed at the entryPoint address parameter
+const entryPointAddress = client.account.getEntryPoint().address;
+const uoHash = await smartAccountClient.sendRawUserOperation({
+  request,
+  entryPoint: entryPointAddress,
+});
+```
+ * 
+ * @param client the smart account client to use for RPC requests
+ * @param args {@link SendTransactionParameters}
+ * @param overrides optional {@link UserOperationOverrides} to use for any of the fields
+ * @param context if the smart account client requires additinoal context for building UOs
+ * @returns a Promise containing the built user operation
+ */
```

Comment:
let's put this on account kit doc (buildUserOperationFromTx or s.t.) and just put a link to that here.

Reply:
I copied from there. I'd rather not link out to to account kit. the goal of the JSDocs is so devs don't have to leave their IDEs for docs